CONTRIBUTING.md

```diff
@@ -40,3 +40,15 @@ Creating very large pull requests may significantly increase review time or even
 
 At this time, we will not be accepting contributions that only fix spelling or grammar errors in documentation, code or
 elsewhere.
+
+### `sncast` Guidelines
+
+#### Command outputs
+
+Please follow these rules when creating outputs for `sncast`:
+
+- Use an imperative tone
+- Keep your message concise and to the point
+- When displaying config, use `key: value` format
+- If the executed command has a natural successor-command, display it as hint in the output
+- Use colored text when displaying fees, addresses and hashes
```

Comment:
Did we agree on the color for these? I'd be worth adding it here

Reply:
We didn't, I think it makes most sense if we temporarily remove this point and bring it back when revamping the outputs because then we will know if chosen colors look good.

Added it todo and mentioned it in https://github.com/foundry-rs/starknet-foundry/issues/2859.